code/src/java/plugin/pretokens/test/PreArmorTypeTester.java

```diff
@@ -56,7 +57,7 @@ public int passes(final Prerequisite prereq, final CharacterDisplay display, CDO
 				{
 
 					String stripped = desiredType.substring(Constants.SUBSTRING_LENGTH_FIVE);
-					StringTokenizer tok = new StringTokenizer(stripped.toUpperCase(), ".");
+					StringTokenizer tok = new StringTokenizer(stripped.toUpperCase(Locale.ENGLISH), ".");
```

Comment:
Types are data controlled
...I think in this case we need to check if caps is reqlly necessary?  Is eq.isType already case insensitive?  Should it be?